default.n2r

```diff
@@ -32,18 +30,16 @@ $BUILD
 %install
 mkdir -p %{buildroot}%{nodejs_sitelib}/%{npm_name}
 $COPYFILES
-# If any binaries are included, symlink them to bindir here
 $BINSNIPPET
 
-%nodejs_symlink_deps
+$SYMLINK
 
 %clean
 rm -rf %{buildroot} %{npm_cache_dir}
```

Comment:
This line should only exist if its bundled and defining npm_cache_dir.

Reply:
Should the buildroot still be removed in ```%clean```? for unbundled? Or is that directory actually inside the buildroot and should the space be removed?

Reply:
I think the default clean is ```rm -rf '%{buildroot}'``` and if we'd place ```%{npm_cache_dir}``` inside the buildroot then it'd automatically clean it up. Do you know if this is correct or am I making things up?